src/chat-bubble/styles.scss

```diff
@@ -0,0 +1,59 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+@use "../../node_modules/@cloudscape-design/design-tokens/index.scss" as cs;
+@use "../internal/shared" as shared;
+
+.root {
+  @include shared.styles-reset;
+
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+
+  color: cs.$color-text-body-default; // to be design token
+  font-family: cs.$font-family-base;
+  font-size: cs.$font-size-body-m; // should we include this? or does the customer define this?
+  line-height: 22px; // same for this one? do we define or let customer define?
+}
+
+.bubble {
```

Comment:
I thought the entire thing, including avatar, is a bubble. Should we call this e.g. "message-area"?